mlir/utils/jenkins/Jenkinsfile

```diff
@@ -508,6 +509,14 @@ pipeline {
                             stage("Copy MIOpen test environment") {
                                 steps {
                                     unstash name: "MIOpen-MLIR-kernels"
+                                    copyArtifacts filter: 'MIOpen/build/MIOpenUserDB/**',\
+                                               optional: true,\
+                                               flatten: true,\
+                                               projectName: "mlir-weekly",\
```

Comment:
Since this bit me before - on public CI, the project might be named MLIR/mlir-weekly and IIRC the perf tests have some workaround for that.

There's also permissions issues that can come up with artifacts. Let's make sure this works before merge.